pkg/process/monitor/process_monitor.go

```diff
@@ -35,6 +36,47 @@ var (
 	}
 )
 
+// processMonitorTelemertry
+type processMonitorTelemertry struct {
+	// process monitor will process :
+	//  o events refer to netlink events received (not only exec and exit)
+	//  o exec process netlink events
+	//  o exit process netlink events
+	//  o restart the netlink connection
+	//
+	//  o refcount is number of used monitor by different subsystem
+	//  o reinitFailed is the number of failed re-initialisation after a netlink restart due to an error
+	//  o processScanFailed would be > 0 if initial process scan failed
+	//  o callbackCalled numbers of callback called
+	events  *telemetry.Metric
+	exec    *telemetry.Metric
+	exit    *telemetry.Metric
+	restart *telemetry.Metric
+
+	refcount          *telemetry.Metric
```

Comment:
wondering why do we need that metric
the number is eventually equals to the number of modules use process monitor (won't during after initialization of the agent) and we know what components are enabled

Reply:
I think we should remove it

Reply:
yeah I removed it as should not hit something bad here